src/javascript/app/base/binary_loader.js

```diff
@@ -101,7 +102,7 @@ const BinaryLoader = (() => {
                     });
             }
         } else if (config.not_authenticated && Client.isLoggedIn()) {
-            BinaryPjax.load(Client.defaultRedirectUrl(), true);
+            handleNotAuthenticated(error_messages.not_authenticated());
```

Comment:
if it's a global object, do we need to pass the error here? why not call it inside `handleNotAuthenticated` itself?

Reply:
We can keep the error messages grouped together so it'll be easier to update them in the future.